app/views/scenarios/index.html.erb

```diff
@@ -21,7 +21,7 @@
         <% @scenarios.each do |scenario| %>
           <tr>
             <td>
-              <%= scenario_label(scenario, content_tag(:i, '', class: 'glyphicon glyphicon-font')) %>
+              <%= scenario_label(scenario, content_tag(:i, '', class: scenario.icon)) %>
```

Comment:
Turns out we can just store `camera` and there is a view helper that will auto generate the necessary `i class`.